uint32 player_id
float32 accel_tgt
float32 steering_tgt